sylvia-derive/src/message.rs

```diff
@@ -682,6 +682,24 @@ impl<'a> MsgVariants<'a> {
                     querier: &'a #sylvia ::cw_std::QuerierWrapper<'a, C>,
                 }
 
+                impl<'a, C: #sylvia ::cw_std::CustomQuery> BoundQuerier<'a, C> {
+                    pub fn querier(&self) -> &'a #sylvia ::cw_std::QuerierWrapper<'a, C> {
+                        self.querier
+                    }
+
+                    pub fn contract(&self) -> &'a #sylvia ::cw_std::Addr {
+                        self.contract
+                    }
+
+                    pub fn new(contract: &'a #sylvia ::cw_std::Addr, querier: &'a #sylvia ::cw_std::QuerierWrapper<'a, C>) -> Self {
+                        Self {contract, querier}
+                    }
```

Comment:
I don't think we ever need this function, but if so, then the signature is bad - `Addr` for this should be taken by owned value.

The reason this is probably needed is that if we store the address in the contract, then we probably store it as `Remote`, and we want to use the `.querier()` function to create a `BoundQuerier`. We use a `borrowed(...)` function only to make a shorthand to `Remote::borrowed(addr).querier(querier)`, but then I would prefer it to be called `borrowed` just like in `Remote`.